### PR TITLE
rulecache.py: make manual rule status boolean

### PR DESCRIPTION
**Pull request by kulikov-a**
Hi!
make manual rules status boolean for comparison results consistency

### CODE REVIEW

File: src/opnsense/scripts/suricata/lib/rulecache.py [hunk 1]
```diff
@@ -123,6 +123,10 @@ def list_local_changes(self):
                     rule_updates[sid] = {'mtime': policy_config_mtime, 'policy_id': None, 'policy': "__manual__"}
                     for rule_item in cnf.items(section):
                         rule_updates[sid][rule_item[0]] = rule_item[1]
+                    status = False
+                    if cnf.get(section, 'enabled') == '1':
+                       status = True
+                    rule_updates[sid]['enabled'] = status
```
Comment (AdSchellevis) [inline src/opnsense/scripts/suricata/lib/rulecache.py:129]: @kulikov-a could be a one-liner:

```
rule_updates[sid]['enabled'] = rule_updates[sid]['enabled'] == '1'
```

Comment (kulikov-a) [reply src/opnsense/scripts/suricata/lib/rulecache.py:129]: @AdSchellevis 
I'll remember ) Thanks!

**Comment by AdSchellevis** [comment]
@kulikov-a thanks!

**Comment by kulikov-a** [comment]
Thanks!!

**Pull request merged**